flybywiresim/a32nx
# feat(hyd): variables for sounds

## Summary of Changes


Adding ptu output required to generate proper sounds for continuous operation


- Power transfer unit can run in burst/bark or in a continuous way

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


To cause continuous operation, try using flaps/slats on yellow pump only.

1. Check  A32NX_HYD_PTU_CONTINUOUS_MODE goes true when ptu is spinning in a continuous manner (more than 1500ms).
PTU rotation can be monitored using A32NX_HYD_PTU_SHAFT_RPM

2. Check High pitch ptu noise on second engine start never happens at the same time as continuous simvar goes true.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
